js/app.js

```diff
@@ -301,7 +301,7 @@
 					this.setRoom(model);
 				}
 			});
-			this._sidebarView.addTab('collections', { label: t('spreed', 'Collections'), icon: 'icon-category-integration' }, this._collectionsView);
+			this._sidebarView.addTab('collections', { label: t('spreed', 'Projects'), icon: 'icon-category-integration' }, this._collectionsView);
```

Comment:
Icon class should be icon-projects once https://github.com/nextcloud/server/pull/14990 is in.

Reply:
Adjusted